libs/langchain/langchain/chains/conversational_retrieval/base.py

```diff
@@ -465,3 +467,59 @@ def from_llm(
             callbacks=callbacks,
             **kwargs,
         )
+
+
+def create_chat_retriever_chain(
```

Comment:
Why `chat` instead of `conversational`?

Reply:
My (perhaps dated) mental model was `chat == ChatModels`, `conversational == chat history`

Reply:
will change to `create_chat_history_retriever`